docs/testing.rst

```diff
@@ -381,6 +381,70 @@ functionality, the AioHTTPTestCase is provided::
    Handles executing an asynchronous function, using
    the :attr:`AioHTTPTestCase.loop` of the :class:`AioHTTPTestCase`.
 
+Patching unittest test cases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Patching test cases is tricky, when using python older than 3.8 ``'patch'`` does not behave as it has to.
+We recommend using asynctest_ that provides a patch method that is capable of creating
+a magic mock that supports async. It can be used with a decorator as well as with a context manager:
+
+.. code-block:: python
+   :emphasize-lines: 1,37,46
```

Comment:
@WisdomPill could you plz check locally that this renders correctly?

Reply:
I just commented with a picture on the above change